usermods/pwm_outputs/usermod_pwm_outputs.h

```diff
@@ -0,0 +1,169 @@
+#pragma once
+#include "wled.h"
+
```

Comment:
Please add #error directive (enclosed in #ifdef) to inform users that usermod is only available for ESP32.

```c++
#ifndef ESP32
  #error This usermod does not support the ESP8266.
#endif
```

Reply:
Good point, made the change